Note - Generating Kiali Session Token (#1366)
rancher/rancher-docs: docs/how-to-guides/advanced-user-guides/istio-setup-guide/generate-and-view-traffic.md

  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/istio-setup-guide/generate-and-view-traffic"/>
</head>

## Prerequisites

> To enable traffic to show up in the graph, ensure you have prometheus installed in the cluster. `Rancher-istio` installs Kiali configured by default to work with the `rancher-monitoring` chart. You can use `rancher-monitoring` or install your own monitoring solution.

Reviewer: ```suggestion
To enable traffic to show up in the graph, ensure that you have Prometheus installed in the cluster. `Rancher-istio` installs Kiali, and configures it by default to work with the `rancher-monitoring` chart. You can use `rancher-monitoring` or install your own monitoring solution.
```

Realize that this is the original text but needed to edit

Author: Applied suggestion

## The Kiali Traffic Graph
To enable traffic to show up in the graph, ensure you have prometheus installed in the cluster. `Rancher-istio` installs Kiali configured by default to work with the `rancher-monitoring` chart. You can use `rancher-monitoring` or install your own monitoring solution.

> The Istio overview page provides a link to the Kiali dashboard. From the Kiali dashboard, you are able to view graphs for each namespace. The Kiali graph provides a powerful way to visualize the topology of your Istio service mesh. It shows you which services communicate with each other.

Reviewer: I agree with making the Prereqs note into its own section, but why get rid of the intro? Users might land on the page from external search.

Reviewer: Ok, I see this was moved below prereqs. IMO it should come earlier to give context for the rest of the page

Author: Moved to the top of the page.

> Additionally, for Istio installations version `103.1.0+up1.19.6` and later, Kiali uses a token value for its authentication strategy. If you are trying to generate or retrieve the token (e.g. for login), note the name of the Kiali service account in Rancher is `kiali`. For more information, refer to the [Kiali token authentication FAQ](https://kiali.io/docs/faq/authentication/).

Reviewer: ```suggestion
Additionally, for Istio installations version `103.1.0+up1.19.6` and later, Kiali uses a token value for its authentication strategy. If you are trying to generate or retrieve the token (e.g. for login), note that the name of the Kiali service account in Rancher is `kiali`. For more information, refer to the [Kiali token authentication FAQ](https://kiali.io/docs/faq/authentication/).
```

Author: Applied suggestion

> Optional: You can change configuration on how data scraping occurs by setting the [Selectors & Scrape Configs](../../../integrations-in-rancher/istio/configuration-options/selectors-and-scrape-configurations.md) options.

Reviewer: ```suggestion
Optional: You can configure in which namespaces data scraping occurs by setting the Helm chart options described in [Selectors & Scrape Configs](../../../integrations-in-rancher/istio/configuration-options/selectors-and-scrape-configurations.md).
```

## The Kiali Traffic Graph

> The Istio overview page provides a link to the Kiali dashboard. From the Kiali dashboard, you are able to view graphs for each namespace. The Kiali graph provides a powerful way to visualize the topology of your Istio service mesh. It shows you which services communicate with each other.

Reviewer: ```suggestion
The Istio overview page provides a link to the Kiali dashboard. From the Kiali dashboard, you can view graphs for each namespace. The Kiali graph provides a powerful way to visualize the topology of your Istio service mesh. It shows you which services communicate with each other.
```

Author: Applied suggestion

> Optional: You can configure in which namespaces data scraping occurs by setting the Helm chart options described in [Selectors & Scrape Configs](../../../integrations-in-rancher/istio/configuration-options/selectors-and-scrape-configurations.md).

Reviewer 2: ```suggestion
Optional: You can configure which namespaces data scraping occurs in by setting the Helm chart options described in [Selectors & Scrape Configs](../../../integrations-in-rancher/istio/configuration-options/selectors-and-scrape-configurations.md).
```

Author: Updated in https://github.com/rancher/rancher-docs/pull/1366/commits/ecfde6184602f6013b055b8c316fb0f02d4faab1

> ### Traffic Visualization

Reviewer 2: ```suggestion
## Traffic Visualization
```

Author: Updated in https://github.com/rancher/rancher-docs/pull/1366/commits/ecfde6184602f6013b055b8c316fb0f02d4faab1, thanks Billy!
